### PR TITLE
feature-5232-Security-Scans-Kics

### PR DESCRIPTION
**Pull request by btme0011**
## What does this PR do?

Run Kics Scan

## Test Plan

Kicks Scan is able to execute the scan successfully but it fails due to following errors [links](https://github.com/btme0011/appwrite/actions/runs/6424134013/job/17444111351). 

As discussed with [steven](https://github.com/appwrite/appwrite/issues/5232#issuecomment-1749800528). Created a seperate issue for that -> [link](https://github.com/appwrite/appwrite/issues/6836)

## Related PRs and Issues

- #5232 

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: .github/workflows/kicks-scan.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  scan:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Scan for misconfigurations
+        uses: Checkmarx/kics-github-action@v1.7.0
+        with:
+          path: ./Dockerfile
```
Comment (stnguyen90) [inline .github/workflows/kicks-scan.yml:17]: So this only scans our Dockerfile? Can we scan the whole repo?

**Review by stnguyen90 — COMMENTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

File: .github/workflows/kicks-scan.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Scan for misconfigurations
```
Comment (stnguyen90) [inline .github/workflows/kicks-scan.yml:1]: Would you please rename the file to scan-misconfigurations.yml?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Comment by tessamero** [comment]
@btme0011 

Thank you for your contribution to Hacktoberfest 2023! We've noticed that your PR is still pending and requires some updates based on our engineering team's feedback.

We would love to see your PR successfully merged and send you the Appwrite swag as a token of appreciation. To remain eligible for the swag, please address the pending suggestions and/or ensure the tests pass  by **Friday, November 17th**. If the PR isn't updated by then, we will unfortunately have to close it due to the end of the Hacktoberfest event.

Looking forward to your updates and thank you!

**Comment by gewenyu99** [comment]
Hey there! There were a lot of big PRs during this Hacktoberfest, and we wanted to give everyone ample time to collaborate with our engineering team. If you were able to merge your PRs during October, amazing. If it’s still not merged, don’t worry about it either. Either way, we’ve got your Hacktoberfest swag minted and ready to ship. 

Please comment with your Discord username here so we can contact you about your shipping information to deliver your Hacktoberfest swag.

**Comment by stnguyen90** [comment]
Closing since we don't need to scan this.

**Pull request closed without merging**